debugger/main.ml

```diff
@@ -229,7 +229,14 @@ let main () =
     if !Parameters.version
     then printf "\tOCaml Debugger version %s@.@." Config.version;
     Loadprinter.init();
-    Load_path.init !default_load_path;
+    let secondary_dirs =
+      if !Clflags.no_std_include then []
```

Comment:
It seems like there should be `Clflags.std_secondary_include_dirs` with this logic.

Reply:
Ah, that's a good idea - so no flag, but the fact it's predicated on `Clflags.no_std_include` does suggest a good home there, yes.